native/package.json

```diff
@@ -11,13 +11,13 @@
     "start:aschaffenburg": "cross-env BUILD_CONFIG_NAME='aschaffenburg' react-native start --reset-cache",
     "start:integreat-e2e": "cross-env BUILD_CONFIG_NAME='integreat-e2e' react-native start --reset-cache",
     "android": "yarn android:integreat-test-cms",
-    "android:integreat-test-cms": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' react-native run-android --no-packager",
-    "android:malte": "cross-env BUILD_CONFIG_NAME='malte' react-native run-android --no-packager",
-    "android:aschaffenburg": "cross-env BUILD_CONFIG_NAME='aschaffenburg' react-native run-android --no-packager",
+    "android:integreat-test-cms": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' react-native run-android --no-packager --appId app.integreat.test",
```

Comment:
Where/why is this appId used/needed?

Reply:
For these builds the app id is different because of the build configs. This fixes the long issue that the app does not start on the device when running these commands

Using constants from the build config would be overkill imo